api/api.yml

```diff
@@ -85,6 +85,110 @@ paths:
       responses:
         '200':
           description: computation stopped
+  /launcher/app-user/applications:
+    get:
+      tags:
+      - launcher
+      summary: Get applications in cockpit
+      responses:
+        '200':
+          description: Applications list
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/ComputationApplication'
+    post:
+      tags:
+      - launcher
+      summary: Add application to cockpit
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ApplicationRequest'
+      responses:
+        '201':
+          description: Application added
+  /launcher/app-user/applications/{id}:
```

Comment:
To samo co wyżej. Czy to jest po to by User wybrał sobie apki które ma mieć na "półce"?
Bo u nas bardziej to widzieliśmy jako wyświetlenie szczegółów aplikacji w celu utworzenia Taska.

Reply:
Dodam jeszcze, że o ile jest UC od dodawania CAppów do Shelfa (nie robimy go), to od usuwania już nie ma.